eth/handler_eth.go

```diff
@@ -187,7 +187,6 @@ func (h *ethHandler) handleBlockBroadcast(peer *eth.Peer, block *types.Block, en
 		log.Warn("Bad Hashes still exist on chain, cannot handle block broadcast yet")
 		return nil
 	}
-
```

Comment:
Please don't touch the file just for a whitespace change

Reply:
Removed it